### DISCUSSION TITLE
Thai (ภาษาไทย)

### DISCUSSION THREAD

**Discussion by rossigee** [sovenor/bitcoin-rocks#14] [Translations]
We need this!

**Comment by sovenor** [comment]
Agreed! Is this something you are willing to help with @rossigee?

**Reply by rossigee** [reply]
Working on a PR here: https://github.com/sovenor/bitcoin-rocks/pull/15

**Reply by sovenor** [reply]
Amazing, thanks!

**Comment by rossigee** [comment]
Ready for further review/merge.

**Comment by sovenor** [comment]
Please note I added a new step to the Translations guide that should be applied to each file:

At the top of the file, change the "last-updated" date to today's date in YYYY-MM-DD format. Change the "locale" to to the [2 letter language code](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) that corresponds to your language. You can also optionally add your name in the list of authors.

Apologies for the oversight! Please make this change on all of the json files.

**Reply by rossigee** [reply]
Done.

**Reply by sovenor** [reply]
Thanks so much! I will review / merge your translations early next week at the latest. Feel free to continue committing translations to that PR in the meantime.

**Comment by sovenor** [comment]
Thai is now available as an option on the website. You may need to clear your cache to see this option. Thank you both for your hard work on this!

@rossigee there are some significant translations needed in index_th.json and common_th.json that would greatly improve the overall experience for Thai visitors. Most of the content in index_th.json (the homepage) is not translated. Is this something you are able to work on?

**Reply by rossigee** [reply]
I deliberately left some of those in English as in my experience I've found that in written Thai they often leave certain brand names and various external references in their original language. I just wanted to get the ball rolling, and I'm hoping once native readers start reading it then they can help identify and fix any gaps to make it read more naturally for them.

**Reply by sovenor** [reply]
That sounds good. Thanks for explaining your reasoning. I don't have any experience with Thai so I will defer to your expertise there. Your work is much appreciated!

**Comment by sovenor** [comment]
Hi, please note that there are 4 new translation strings that are now in "common_th.json"

"common_self_custody": "SELF-CUSTODY",
"common_not_your_keys": "NOT YOUR KEYS",
"common_cold_wallet": "COLD WALLET",
"common_hot_wallet": "HOT WALLET"

You should see these at the end of the file when you pull the latest changes. Thanks!

**Comment by sovenor** [comment]
Two new translation strings have been added to the top of common_th.json:

"common_footer_mission_1"
"common_footer_mission_2"

Please translate these when you are able. Thank you!

**Comment by sovenor** [comment]
New translation strings have been added to inflation_th.json:

Line 11: "inflation_sign_got_inflation"
Line 12: "inflation_save_in_bitcoin"
Line 42: "inflation_usd_but_why"

There are also 2 new pages that don't necessarily need translation since the signs campaign is USA only:

signs_th.json
sign-success_th.json

Thanks for your continued contributions!

**Reply by rossigee** [reply]
https://github.com/sovenor/bitcoin-rocks/pull/50

**Reply by sovenor** [reply]
Thanks for your time on this and patience waiting for approval! Just approved and merged the changes.

**Comment by sovenor** [comment]
Translation strings have been modified slightly in wallets_th.json:

Line 11: "two simple questions" is now "these simple questions"
Line 13: "two simple questions" is now "these simple questions"

These new strings have also been added to the same file:

Lines 33 - 39 to add a section about recovery phrases / seed phrases.

Please translate these if you have some spare time. Thank you!

**Comment by Ployten** [comment]
Hello, nice to meet you. I am a Thai native speaker. Hope I can contribute something here.

**Reply by sovenor** [reply]
Let me know if you have any questions!

**Comment by sovenor** [comment]
Just wanted to make a note here that Thai is available across the entire website, and has been for a while thanks to @rossigee!
